Purchasing.Core/Helpers/StringExtensions.cs

```diff
@@ -6,5 +6,10 @@ public static string Summarize(this string str, int maxLength = 25)
         {
             return str.Length < maxLength ? str : string.Format("{0}...", str.Substring(0, maxLength - 3));
         }
+
+        public static string NullCheck(this string str)
+        {
+            return str ?? string.Empty;
+        }
```

Comment:
I'm okay with using this method, though I don't think it adds much value vs just doing `?? string.Empty`. If we keep it, it might be more clear to name it something like `DefaultToEmpty()` or `EmptyIfDefault()`.

Reply:
I'm ok with changing it. Just all the null checks here which weren't needed before were bugging me here:
`acell2.Add(new Paragraph(order.Vendor == null ? string.Empty : order.Vendor.Line1.NullCheck()).SetFont(_font));`